lib/cocoapods-core/specification/dsl.rb

```diff
@@ -1259,6 +1259,31 @@ def dependency=(args)
 
       #------------------#
 
+      # @!method project_header_files=(project_header_files)
+      #
+      #   A list of file patterns that should be used to mark project headers.
+      #
+      #   ---
+      #
+      #   These patterns are matched against the public headers (or all the
+      #   headers if no public headers have been specified) to exclude those
+      #   headers which should not be exposed to the user project and which
+      #   should not be used to generate the documentation.
+      #
```

Comment:
Does the `private_header_files` documentation below need to be updated?  Or should the `private_header_files`  and `project_header_files` specifications never overlap?

Reply:
I dont think it needs to be updated. I think what we need to specify is that project headers are not copied in the build directory like public or private headers do. Project headers are truly "private" despite the weird naming.

Reply:
I see what you are saying though I will update it a bit.